module/platform/Webots/src/Webots.cpp

```diff
@@ -242,6 +247,86 @@ namespace module::platform {
             setup_connection();
         });
 
+
+        on<Configuration>("camera.yaml").then([this](const Configuration& config) {
+            // std::string serial_number = config["serial_number"].as<std::string>();
+            // Strip the .yaml off the name of the file to get the name of the camera
+            std::string name = ::basename(config.fileName.substr(0, config.fileName.find_last_of('.')).c_str());
+
+            // // Add camera to list.
+            // auto it = cameras
+            //               .insert(std::make_pair(serial_number,
+            //                                      CameraContext{
+            //                                          *this,
+            //                                          name,
+            //                                          num_cameras++,
+            //                                          Image::Lens(),      // Lens is constructed in settings
+            //                                          Eigen::Affine3d(),  // Hpc is set in settings
+            //                                      }))
+            //               .first;
```

Comment:
```suggestion
            // Strip the .yaml off the name of the file to get the name of the camera
            std::string name = ::basename(config.fileName.substr(0, config.fileName.find_last_of('.')).c_str());
```

Should probably remove the commented out code?

Reply:
Seems like a good idea